bundles/org.openhab.binding.daikin/src/main/java/org/openhab/binding/daikin/internal/api/InfoParser.java

```diff
@@ -92,8 +93,23 @@ public static String urldecode(String value) {
         try {
             return URLDecoder.decode(value, StandardCharsets.UTF_8.toString());
         } catch (UnsupportedEncodingException e) {
-            logger.warn("Unsupported encoding error in '{}'", value, e);
+            LOGGER.warn("Unsupported encoding error in '{}'", value, e);
             return value;
         }
     }
+
+    /**
+     * This method combines different direction parameters to one.
+     * 
+     * @param ud up/ down value S or 0
+     * @param lr left/ right value S or 0
+     * @return combined int value based on {@link FanMovement} enum
+     */
+    public static int parseSpecial(String ud, String lr) {
```

Comment:
Please call this `parseFanMovement`, and move it into ControlInfo.java because it's only specific to the `ac_unit` thing and not to `airbase_ac_unit` thing.

Reply:
renamed and moved.